{"name": "TC01: Login to Grafana", "status": "broken", "statusDetails": {"message": "AttributeError: module 'appium.webdriver' has no attribute 'Chrome'", "trace": "request = <SubRequest 'init_web_driver' for <Function test_verify_login>>\n\n    @pytest.fixture(scope=\"class\")\n    def init_web_driver(request):\n        if get_data(\"ExecuteApplitools\").lower() == 'yes':\n            globals()['driver'] = get_web_driver()\n        else:\n>           edriver = get_web_driver()\n\nconftest.py:44: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nconftest.py:85: in get_web_driver\n    driver = get_chrome()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\n    def get_chrome():\n>       chrome_driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()))  # Selenium 4.x\nE       AttributeError: module 'appium.webdriver' has no attribute 'Chrome'\n\nconftest.py:108: AttributeError"}, "description": "Verify a successful login to Grafana", "start": 1677613396079, "stop": 1677613396079, "uuid": "93df11f5-83d0-44be-8ec1-09a031d16835", "historyId": "54e61a4397564b26acdf4b5fe208f939", "testCaseId": "4727d0277c9099c0888dae150522b402", "fullName": "test_cases.test_web.TestWeb#test_verify_login", "labels": [{"name": "tag", "value": "run_this"}, {"name": "tag", "value": "sanity"}, {"name": "tag", "value": "@pytest.mark.usefixtures('init_web_driver')"}, {"name": "parentSuite", "value": "test_cases"}, {"name": "suite", "value": "test_web"}, {"name": "subSuite", "value": "TestWeb"}, {"name": "host", "value": "BarLaptop"}, {"name": "thread", "value": "10244-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.test_web"}]}